bin/verdicts.py

```diff
@@ -0,0 +1,394 @@
+from pathlib import Path
+import shutil
+import sys
+from enum import Enum
+
+from util import ProgressBar
+import config
+from colorama import Fore, Style
+
+
+class Verdict(Enum):
+    """The verdict of a testcase or testgroup"""
+
+    ACCEPTED = 1
+    WRONG_ANSWER = 2
+    TIME_LIMIT_EXCEEDED = 3
+    RUN_TIME_ERROR = 4
+
+
+def to_char(v: Verdict | None):
```

Comment:
can we add a flag to get small letters instead and then remove `VerdictTable.colors` and update `VerdictTable._get_verdict` ?

Reply:
That’s the direction I want this to go in. But let me hold back a bit before introducing too many dependencies.